scripts/lib/upgrade-zulip-stage-2

```diff
@@ -35,6 +35,17 @@ logging.Formatter.converter = time.gmtime
 logging.basicConfig(format="%(asctime)s upgrade-zulip-stage-2: %(message)s",
                     level=logging.INFO)
 
+
+def unsupported_os():
```

Comment:
Maybe call this `error_unsupported_os`, to be clear it's not `if_unsupported_os`.

Reply:
Err, better, `error_desupported_os`.

Reply:
We also call it if it has never been supported -- i.e. I think we hit this path if you're on Solaris or whatnot.  `desupported` seems inaccurate for that.

Reply:
Ah, I see we catch that in straight-up `install`.  Will rename.